### ISSUE TITLE
[Feature Request] Import timecodes from SRT / external file

### ISSUE THREAD

**Issue by vevv** [MonoS/SupMover#25]
Subtitle Edit has such an option, but it's a non-ideal tool for editing any kind of PGS subtitles for a lot of reasons. This would make syncing much easier, since you could use any tool on a blank SRT and then apply it here.

**Comment by MonoS** [comment]
yeah, it should be feasible, I'll try working on it, have you any suggestion for the name? I was thinking about `timefrom` or `timecode_from`, what do you think?
